.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,7 @@
 ### What was wrong?
 
-Related to Issue #
+Closes issue #
 
 ### How was it fixed?
+
+@gnosis/safe-services
```

Comment:
nitpick: did you want to change this in this PR?

Reply:
Yes, I didn't want to create a new PR for this 😅